packages/components/atoms/f-button/stories/RouterButton.stories.js

```diff
@@ -0,0 +1,79 @@
+import Vue from 'vue';
+import Router from 'vue-router';
+
+import {
+    withKnobs,
+    select,
+    boolean,
+    text
+} from '@storybook/addon-knobs';
+import { withA11y } from '@storybook/addon-a11y';
+import FButton from '../src/components/Button.vue';
+
+Vue.use(Router);
+
+const Foo = { template: '<div>foo</div>' };
```

Comment:
Does this just load in some example routes? If so, I'd be inclined to use `Example Route 1` & `Example Route 2` rather than `Foo` and `Bar` for the content.

Reply:
Updated accordingly